src/Symfony/Component/Translation/Command/XliffLintCommand.php

```diff
@@ -223,18 +228,29 @@ private function getFiles(string $fileOrDirectory)
         }
     }
 
-    private function getStdin(): ?string
+    private function getStdin(SymfonyStyle $io): string
     {
-        if (0 !== ftell(STDIN)) {
-            return null;
-        }
-
-        $inputs = '';
-        while (!feof(STDIN)) {
-            $inputs .= fread(STDIN, 1024);
+        $xliff = '';
+        if (0 === ftell(STDIN)) {
+            while (!feof(STDIN)) {
+                $xliff .= fread(STDIN, 1024);
+            }
+        } else {
+            $io->block('Write the Xliff content (press <return> twice to stop)', null, 'fg=black;bg=yellow', ' ', true);
```

Comment:
I don't think this works. At least I know no other Unix command that has a similar behavior. To me, the ftell() call relies on a race condition that is fragile by nature. It should be removed in 5. And we cannot display this message nor detect EOF using the empty line. There's already CTRL+D to do so.